### PR TITLE
Minor text fixes

### PR DESCRIPTION
**Pull request by imapokemon4**
Just a few things I missed in #379.

### CODE REVIEW

**Comment by imapokemon4** [comment]
I'm not sure why there are 4 commits shown.

Won't only 85a30c5 be merged?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -141,7 +139,7 @@
     <string name="iv">IV</string>
     <string name="percent">%d%%</string>
     <string name="unknown_percent">"?%"</string>
-    <string name="percent_phrase">Percent phrase</string>
-    <string name="iv_range_phrase">IV range phrase</string>
-    <string name="higher_values">Higher values</string>
+    <string name="percent_phrase">Overall Appraisal:</string>
+    <string name="iv_range_phrase">Best Stat(s) Appraisal:</string>
+    <string name="higher_values">Best Stat(s):</string>
```
Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: I think percent_phrase, iv_range_phrase and higher_values are unused currently... gotta check.

**Comment by Blaisorblade** [comment]
The parent commit of 85a30c5cedf88853a976d4b1ea289de2dd014308 is *not* a commit in our fork.

I squashed your last PR and I'd squash this one too, so this won't affect us. The squashing is one reason why f213a4e852914782497bd220674df132f8a23525 is not part of the history of the main fork.

If you want to fix that, once the PR is merged, you should ensure your branch is rebased onto master of the main fork.

**Comment by imapokemon4** [comment]
Magic!

**Comment by imapokemon4** [comment]
@Blaisorblade 

Anything else I need to do get this merged?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -141,7 +139,4 @@
     <string name="iv">IV</string>
     <string name="percent">%d%%</string>
     <string name="unknown_percent">"?%"</string>
-    <string name="percent_phrase">Percent phrase</string>
-    <string name="iv_range_phrase">IV range phrase</string>
-    <string name="higher_values">Higher values</string>
```
Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: Just double-checked this should go, good.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -124,8 +122,8 @@
     <string name="attack">Attack</string>
     <string name="defense">Defense</string>
     <string name="stamina_hp">HP</string>
-    <string name="missing_inputs">Missing data. Please fill CP and HP inputs.</string>
-    <string name="ivtext_no_possibilities">There are no IV possible combinations for this input. Please double check the information.</string>
+    <string name="missing_inputs">CP and HP are required.</string>
+    <string name="ivtext_no_possibilities">There are no possible combinations. Make sure the values you have entered are correct.</string>
```
Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: I'm not sure of these changes. Why?
I'm tempted to go with "Please fill in CP and HP." (more direct) and "No possible IV combinations. Make sure your inputs are correct".

Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: BTW I could also merge as-is and revise this later if you don't have objections to the text I propose.

And sorry for not replying earlier, was just slightly in a hurry and unsure of the best text.

Comment (imapokemon4) [inline app/src/main/res/values/strings.xml]: In my opinion (of course):

Compared to "Missing data.", "CP and HP are required" is more to the point and at the same time implies that the user will need to fill them in.

"IV" in our case is implied as the app is an IV calculator. So I removed that.

Compared to "for this input. Please double check the information.", "Make sure the values you have entered are correct." is more concise and clear.

Comment (imapokemon4) [inline app/src/main/res/values/strings.xml]: In my opinion (again, of course):

"Make sure your inputs are correct" doesn't sound as professional.

Comment (imapokemon4) [inline app/src/main/res/values/strings.xml]: But, of course, I respect your opinion and will be perfectly fine with whatever you choose to do. :)

Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: Thanks for your comments, I'll keep them in mind.

> "Make sure your inputs are correct" doesn't sound as professional.

I think I know what you mean, but I've been trained out of that style in my writing courses, following the tradition set for instance by George Orwell.

Comment (imapokemon4) [inline app/src/main/res/values/strings.xml]: I think my dispute is more with the word "inputs" rather than anything George Orwell wrote.

Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: Looked it up, "input" might be uncountable in this context 😫 —would it be better "Make sure your input is correct"?

**Comment by Blaisorblade** [comment]
Meanwhile, merging your PR.

**Comment by imapokemon4** [comment]
To me "input" does in fact sound better than "inputs". However to the end user I believe "the values you have entered" will be more clear.

Thanks for merging!

**Pull request merged**